### ISSUE TITLE
Headless mode fails due to incorrect processing of CDP user agent message

### ISSUE THREAD

**Issue by slimshreydy** [stephanlensky/zendriver#23]
I am trying to use zendriver in a simple AWS-based debian box (via [Modal](https://modal.com)) with headless Chrome. I installed the `chromium` package via `apt-get` and then tried running the following code in the Python interpreter:

```
import zendriver as uc

async def open_page():
    browser = await uc.start(sandbox=False, headless=True)
    page = await browser.get('https://www.google.com')
    print(await page.get_content())
    await browser.stop()

uc.loop().run_until_complete(open_page())
``` 

This yields an `AttributeError` saying `'tuple' object has no attribute 'value'` in the `_prepare_headless` call:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.11/asyncio/base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "<stdin>", line 3, in open_page
  File "/usr/local/lib/python3.11/site-packages/zendriver/core/browser.py", line 271, in get
    frame_id, loader_id, *_ = await connection.send(cdp.page.navigate(url))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/zendriver/core/connection.py", line 422, in send
    await self._prepare_headless()
  File "/usr/local/lib/python3.11/site-packages/zendriver/core/connection.py", line 509, in _prepare_headless
    if response and response.value:
                    ^^^^^^^^^^^^^^
AttributeError: 'tuple' object has no attribute 'value'
```

This `response` object is coming from a CDP call where zendriver tries to get the userAgent:

```
response = await self._send_oneshot(
    cdp.runtime.evaluate(
        expression="navigator.userAgent",
        user_gesture=True,
        await_promise=True,
        return_by_value=True,
        allow_unsafe_eval_blocked_by_csp=True,
    )
)
```

It appears that this response object is actually a tuple that looks like this: `(RemoteObject(type_='string', subtype=None, class_name=None, value='Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/120.0.0.0 Safari/537.36', unserializable_value=None, description=None, deep_serialized_value=None, object_id=None, preview=None, custom_preview=None), None)`

The value `_prepare_headless` is trying to extract is hence not at `response.value`, but rather `response[0].value`. I am making a PR to fix this.

# Error profile

- `zendriver` version: `zendriver==0.2.0`
- OS/platform: `Debian GNU/Linux 11 (bullseye)`
- Linux kernel version: `4.4.0`
- Python version: `3.11.0`
- Chrome version: `120.0.6099.224`
- Can you reliably reproduce the issue?: Yes, see above.

**Comment by slimshreydy** [comment]
Also just wanted to thank you for making this library! Besides this one issue, it worked like a charm -- `nodriver` was failing left and right in my environment.

**Comment by stephanlensky** [comment]
Thanks, I'm glad the library has helped you 🙂 

This does look like a bug, I'll take a look at the PR shortly.

**Comment by stephanlensky** [comment]
BTW - I appreciate all of the detail you added to this issue, it's very helpful.

**Comment by stephanlensky** [comment]
Confirmed I am seeing the same on my Debian machine.

**Comment by stephanlensky** [comment]
This is fixed starting in version 0.2.2 (https://github.com/stephanlensky/zendriver/pull/24).
